tests/components/tradfri/test_diagnostics.py

```diff
@@ -1,15 +1,33 @@
 """Tests for Tradfri diagnostics."""
-from unittest.mock import MagicMock, Mock
+from unittest.mock import MagicMock, Mock, PropertyMock, patch
+
+import pytest
 
 from homeassistant.core import HomeAssistant
 
 from .common import setup_integration
-from .test_fan import mock_fan
+from .test_sensor import mock_fan
 
 from tests.components.diagnostics import get_diagnostics_for_config_entry
 from tests.typing import ClientSessionGenerator
 
 
+@pytest.fixture(autouse=True)
+def setup(request):
```

Comment:
This is moved for now so we can refactor the diagnostics tests later in a separate PR.